includes/nets-easy-functions.php

```diff
@@ -204,6 +204,12 @@ function wc_dibs_confirm_dibs_order( $order_id ) {
 	if ( null === $payment_id ) {
 		$payment_id = WC()->session->get( 'dibs_payment_id' );
 	}
+
+	if ( empty( $payment_id ) ) {
+		Nets_Easy_Logger::log( 'No payment ID found in the confirmation request.' );
```

Comment:
See above comment about identifying log entry.

Reply:
Sorry, I should have been clearer. The identifying needs to have been logged sometime earlier in an API request (e.g., create, update). In this, the WooCommerce session ID isn't associated with a Ledyer session or WooCommerce order, it will therefore be difficult to figure out which one of these it refers to. Sometimes, it is just not possible to find an identifier, e.g., in a create session request.

Reply:
Like this? :) I've tried making them a bit more descriptive/user friendly as well.